### ISSUE TITLE
How do you enable macros?

### ISSUE THREAD

**Issue by voor** [mojohaus/rpm-maven-plugin#77]
I wanted to follow the example for scriptlets located here:
https://fedoraproject.org/wiki/Packaging:Scriptlets

That requires you to add: 
```
%{?systemd_requires}
BuildRequires: systemd
```
To the RPM spec.

It guess this becomes a two part question, how do you add `BuildRequires` and how do you then add an arbitrary macro like `%{?systemd_requires}`?

**Comment by JeremyVV** [comment]
I am hoping this gets added soon too.

**Comment by sisivy** [comment]
any update on this issue? When do you plan to add it?

**Comment by voor** [comment]
Yes, (although I don't work at that company anymore) I no longer incorporated building the RPMs into the Maven build cycle, and instead would fire off a secondary build for the RPMs afterwards.  Gave me the full flexibility configuring all RPM build options.
